compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirMemberPropertyChecker.kt

```diff
@@ -111,11 +111,11 @@ object FirMemberPropertyChecker : FirRegularClassChecker() {
     private fun checkAccessor(
         accessor: FirPropertyAccessor?,
         delegate: FirExpression?,
-        report: (FirSourceElement, FirPropertyAccessorSymbol) -> Unit,
+        report: (FirSourceElement, FirPropertyAccessorSymbol, hasCustomBody: Boolean) -> Unit,
     ) {
-        if (accessor != null && accessor !is FirDefaultPropertyAccessor && accessor.hasBody && delegate == null) {
+        if (accessor != null && delegate == null) {
             accessor.source?.let {
-                report.invoke(it, accessor.symbol)
+                report.invoke(it, accessor.symbol, accessor !is FirDefaultPropertyAccessor && accessor.hasBody)
```

Comment:
Duh, it was me writing this condition, but could you try just `accessor.hasBody` without default accessor check?

Reply:
Sure. I wasn't sure if that type check is really needed. Looks like tests are passing without it from local runs